x/incentives/keeper/gauge.go

```diff
@@ -127,9 +127,41 @@ func (k Keeper) CreateGauge(ctx sdk.Context, isPerpetual bool, owner sdk.AccAddr
 		return 0, types.ErrZeroNumEpochsPaidOver
 	}
 
+	// If the gauge has no lock, then we assume it is a concentrated pool and ensure
+	// the gauge "lock" duration is an authorized uptime.
+	isConcentratedPoolGauge := distrTo.LockQueryType == lockuptypes.NoLock
```

Comment:
One concern that we should keep in mind is what if we delay releasing off-chain incentives and we need to incentivize a new pool type (e.g. cosmwasm) with a `NoLock` gauge.

For that reason, it might be best to avoid connecting `NoLock` gauge abstraction to a CL pool.

Reply:
Unfortunately, the abstraction is already intimately connected with CL. This is why, for instance, we added a new `ByGroup` type for group gauges. In the future, I think it would make sense to rename this to `Concentrated` for better readability, as if we add any other "NoLock" gauges e.g. for cosmwasm pools, it would almost certainly need to be on a new type.

Reply:
Not a strong comment but I'm in disagreement. If we wanted to extend `NoLock` to also support CW pools, for example, we could achieve that by doing a short refactor to the `distributeInternal` function. The refactor would entail checking the type of the pool that `NoLock` gauge incentivizes and handling the distribution accordingly.

To be clear, the core of the change I'm proposing is renaming identifiers. For example, `isInternalConcentratedPoolGauge` to `isInternalNoLockGauge`.

If disagreement, happy to not over-index on this so feel free to resolve

Reply:
Sounds good. Happy to make the variable name change for now, as I do agree it's more consistent with our existing naming convention.